test/client.test.js

```diff
@@ -3231,7 +3231,10 @@ describe('ACC Client', function () {
                     method: 'POST',
                     processData: false,
                     credentials: 'include',
```

Comment:
Can we remove this "credentials: include" line (from both methods) as well as the corresponding "credentials: options.credentials || 'same-origin' " in transport.js?

Reply:
Good point. Let me check if that is needed at all.